src/services/print.js

```diff
@@ -488,6 +492,49 @@ ngeo.Print.prototype.encodeVectorStyleStroke_ =
 
 
 /**
+ * @param {Array.<MapFishPrintSymbolizer>} symbolizers Array of MapFish Print
+ *     symbolizers.
+ * @param {!ol.style.Text} textStyle Text style.
+ * @private
+ */
+ngeo.Print.prototype.encodeTextStyle_ = function(symbolizers, textStyle) {
+  var symbolizer = {
+    type: 'Text'
+  };
```

Comment:
I don't know. I have no particular experience with closure. If you give me a hit, I may be able to write it.